util/dashboard/gen_dashboard_entry.py

```diff
@@ -229,12 +216,13 @@ def print_version1_format(obj, outfile):
     name = html.escape(obj['name'])
     genout(outfile, "        <td class=\"fixleft\">" +
                     get_linked_design_spec(obj) + "</td>\n")
-    genout(outfile, "        <td class=\"hw-stage\">" +
+    genout(outfile, "        <td class=\"dv-plan\">" +
                     get_linked_dv_plan(obj) + "</td>\n")
-    genout(outfile, "        <td class=\"hw-stage\">" +
+    genout(outfile, "        <td class=\"version\">" +
                     get_linked_version(obj) + "</td>\n")
-    genout(outfile, "        <td class=\"hw-stage\"><span class='hw-stage'>" +
-                    get_development_stage(obj, obj) + "</span></td>\n")
+
+    for stage_html in get_development_stage(obj, obj):
+        genout(outfile, "        <td class=\"hw-stage\"><span class='hw-stage'>" + stage_html + "</span></td>\n")
```

Comment:
nit: this probably exceed 80 chars / line.

Reply:
I did a minor code reformat, to appease `flake8`.